data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+		weapon
+			"blast radius" 12
+			"shield damage" 240
+			"hull damage" 150
+			"hit force" 120
+	outfits
+		"Speckle Coilgun"
+		"Speck Round" 900
+	
+		"Concentrating Solar Array"
+		"Ultracapacitor Cell"
+		"Multispectral Scanner"
+	
+		"Magnetoplasma Drive"
+
+	engine 0 27
+	"reverse engine" -5 -12
+		zoom 0.5
+	"reverse engine" 5 -12
+		zoom 0.5
+	"steering engine" -5 -12
+		angle 90
+		zoom 0.5
+		right
+	"steering engine" 5 -12
+		angle -90
+		zoom 0.5
+		left
+	"steering engine" -3 24
+		angle 90
+		zoom 0.5
+		left
+	"steering engine" 3 24
+		angle -90
+		zoom 0.5
+		right
+	gun 0 -27 "Speckle Coilgun"
+	gun 0 -27
+
+	explode "tiny explosion" 9
+
+	description "The Vibratia Interdictor is the primary Avgi strike fighter. While individually fragile, Vibratia are usually deployed in large swarms, and are agile enough to be surprisingly survivable even under heavy fire."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+
+ship "Undulon"
+	sprite "ship/avgi undulon"
+	thumbnail "thumbnail/avgi undulon"
+	attributes
+		category "Drone"
+		licenses
+			"Avgi Atomics"
+		"cost" 36000
+		"hull" 600
+		"automaton" 1
+		"crew equivalent" 1
+		"mass" 60
+		"drag" 15
+		"heat dissipation" .45
+		"fuel capacity" 60
+		"outfit space" 84
+		"weapon capacity" 42
+		"engine capacity" 18
+		weapon
+			"blast radius" 12
+			"shield damage" 180
+			"hull damage" 120
+			"hit force" 90
+	outfits
+		"External Launch Tube"
+		`"Orchid" Nuclear Missile` 6
+
+		"Concentrating Solar Cell" 3
+		"Ultracapacitor Cell"
+		"Multispectral Scanner"
+
+		"Magnetoplasma Drive"
+
+	engine -15 3
+		angle 15
+		zoom 0.5
+	engine 15 3
+		angle -15
+		zoom 0.5
+	"reverse engine" -15 -18
+		angle -15
+		zoom 0.5
+	"reverse engine" 15 -18
+		angle 15
+		zoom 0.5
+	"steering engine" -15 -18
+		angle 165
+		zoom 0.5
+		left
+	"steering engine" 15 -18
+		angle 195
+		zoom 0.5
+		right
+	"steering engine" -15 3
+		angle 15
+		zoom 0.5
+		right
+	"steering engine" 15 3
+		angle -15
+		zoom 0.5
+		left
+
+	gun 0 -29 "External Launch Tube"
+
+	explode "tiny explosion" 12
+
+	description "The Undulon Orbital Silo was first envisioned as a cheap static defense satellite, little more than a tube capable of storing and launching nuclear-tipped missiles. Over time, this specialization was broadened, and the modern line of Undulons is capable of both mounting a large fixed gun and carrying the energy supply needed to support it. Their mostly hollow construction provides a great deal of internal space for their size, but also makes the drone quite fragile."
+	description "Undulons are usually seen defending critical planets and stations, ready to unload their atomic payloads on any hostile ships that jump into the system."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+ship "Optikon"
```

Comment:
Integrated scanners betweeen them provide the equivalent of about 18 space, bringing the Optikon up to an effective outfit space of 78, which seems fine for a drone.